### ISSUE TITLE
Layout 2 columns of equal width are wrapping in the demo

### ISSUE THREAD

**Issue by asacarter** [Shopify/polaris-react#2675]
## Issue summary

On the layout page of the Polaris website, the 2 columns of equal width are wrapping.

https://polaris.shopify.com/components/structure/layout

## Expected behavior

They should be displayed side by side.

## Actual behavior

They are wrapping.

**Comment by AndrewMusgrave** [comment]
Hi @asacarter , thanks bringing this up! This is because the view area in the iframe is very small. If you open the example in code sandbox they'll display side by side 😄

**Comment by asacarter** [comment]
That's fair enough. But then why are the 3 equal layouts displayed side by side?

If think the half and third layouts need a bit of work at various breakpoints as I'm getting some weird behaviour on a layout I'm creating. I ended up having to overwrite the default values to get it to work correctly at smaller sizes.

I have just created a layout of three columns with two blocks of identical content in each.

In a wide viewport, the layout is correct:

```
1 2 3
4 5 6
```
Between 540px - 768px and 780px - 1063px, I get a layout like:
```
1 2 
4 5
3 3  
6 6
```
with only 3 and 6 using 100% width. I would expect each element to be 100% width.
